source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
+   collection.find().sort(ascending("_id")).into(results);
+   for (Document result : results) {
+         System.out.println(result.toJson());
+   }
+
+The output of the code example above should look something like this: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   ...
+
+Descending
+~~~~~~~~~~
+
+To specify a descending sort, use the ``Sorts.descending()`` static factory
+method. Pass the ``Sorts.descending()`` method the name of the field you need to sort in descending order.
+
+The following code snippet shows how to specify a descending sort on the
+``_id`` field: 
 
 .. code-block:: java
+   
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("_id"));
+
+
+The code snippet above returns the documents in the
+:ref:`sample collection <sorts-crud-sort-sample>`  
+in descending order: 
+
+.. code-block:: json
 
-    Document{{_id=1, title=The Brothers Karamazov, length=824, author=Dostoyevsky}}
-    Document{{_id=5, title=Cryptonomicon, length=918, author=Stephenson}}
-    Document{{_id=3, title=Atlas Shrugged, length=1088, author=Rand}}
-    Document{{_id=6, title=A Dance with Dragons, length=1104, author=Martin}}
-    Document{{_id=4, title=Infinite Jest, length=1104, author=Wallace}}
-    Document{{_id=2, title=Les Misérables, length=1462, author=Hugo}}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   ...
 
-You can also sort by multiple fields in different orders, using ``Sorts.orderBy()``. 
-Consider this example where you first sort on ``length`` in an ascending order, and then on
-``title`` in a descending order:
+.. _sorts-crud-handling-ties:
+
+Handling Ties
+~~~~~~~~~~~~~
+
+A tie occurs when two or more documents have a field with identical values.
+MongoDB does not guarantee sort order in the event of ties. For example, suppose
+we encounter a tie when applying a sort to the
+:ref:`sample collection <sorts-crud-sort-sample>` using the following
+code:
 
 .. code-block:: java
-  :emphasize-lines: 7
-    
-    import static com.mongodb.client.model.Sorts.ascending;
-    import static com.mongodb.client.model.Sorts.descending;
-    import static com.mongodb.client.model.Sorts.orderBy;
 
-    // sort first by length ascending, then title descending
-    MongoCursor<Document> cursor = collection.find()
-        .sort(orderBy(ascending("length"), descending("title")))
-        .iterator();
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter"));
+
+Since multiple documents contain the value "a" for the field on which we perform the sort,
```

Comment:
Would it help for this to be more specific about what the "multiple documents" are? Since you're calling an operation, maybe they could be "documents that matched the query" or "results of the query" or other.

Reply:
Thanks for the note. I agree, "multiple documents" is unproductively vague. I have added clarifying language. fix: https://github.com/mongodb/docs-java/pull/66/commits/0d419400d1da8dcf44314d7a571718bf2e6fe068

Going forward, I will try to keep an eye for these types of unclear subjects. "Multiple Documents" is too abstract.